sdk/spring/spring-cloud-azure-stream-binder-eventhubs-core/src/main/java/com/azure/spring/cloud/stream/binder/eventhubs/properties/EventHubsConsumerProperties.java

```diff
@@ -23,6 +23,12 @@
 
     private final CheckpointConfig checkpoint = new CheckpointConfig();
 
+    /**
+     * Get checkpoint.
```

Comment:
Let's write it as "Get the checkpoint config" or "Get the {@link CheckpointConfig}"